src/modules/deposit/entry-point/http/dto.ts

```diff
@@ -130,6 +130,32 @@ export class GetDepositsV2Query {
   @IsOptional()
   @IsString()
   @Type(() => String)
-  @ApiProperty({ example: "0x", required: true })
+  @ApiProperty({ example: "0x", required: false })
   tokenAddress: string;
 }
+
+export class GetDepositsForTxPageQuery extends GetDepositsV2Query {
+  @IsOptional()
+  @IsString()
+  @Type(() => String)
+  @ApiProperty({ example: "0x", required: false })
+  userAddress: string;
+
+  @IsOptional()
+  @IsBoolean()
+  @Type(() => Boolean)
+  @ApiProperty({ example: "true", required: false })
+  skipOldUnprofitable: boolean;
+
+  @IsOptional()
+  @IsEnum(
+    {
+      STATUS: "status",
+    },
+    {
+      message: "Must be one of: 'status'",
+    },
+  )
```

Comment:
QQ: where do we specify the enum?

Reply:
It gets inferred by the `@IsEnum` decorator AFAICT

Reply:
Right, but where do we define the enum in the code? Unless the enum is defined in-line with:
```
   {
      STATUS: "status",
    },
```